overlord/snapstate/handlers.go

```diff
@@ -3743,6 +3926,13 @@ func (m *SnapManager) doConditionalAutoRefresh(t *state.Task, tomb *tomb.Tomb) e
 
 	tss, err := autoRefreshPhase2(context.TODO(), st, snaps, t.Change().ID())
 	if err != nil {
+		if errors.Is(err, &BusySnapError{}) {
```

Comment:
this looks strange, as this is about many snaps? don't we need to potentially create two changes?